utils/wasm-instrument/src/rules.rs

```diff
@@ -0,0 +1,154 @@
+use core::num::NonZeroU32;
+use wasm_instrument::{
+    gas_metering::{MemoryGrowCost, Rules},
+    parity_wasm::elements::{self, Instruction},
+};
+
+pub struct CustomConstantCostRules {
```

Comment:
this looks like a code duplication however the rules need to be in sync with the `Schedule`. Although it would require a bit of `Schedule` refactoring it doesn't worth it IMO.

So at the very least the unit test in `Schedule` crate/module could be written that both `instruction_cost` impls return `Some`/None respectively for the specific instruction(s).

Reply:
@gshep should be fine now